src/components/main/index.tsx

```diff
@@ -1,50 +1,64 @@
 import React from 'react';
+import { useRouter } from 'next/router';
 
 import HomeLayout from '../common/layout/layout';
 import Beer from '../common/beer/beer';
 import PreferenesModal from './preferenes-pop-up';
-
 import { MainContainer, MainContent, MainTab, Text, TabButton } from './styled';
 
-import Data, { DummyProps } from './dummy';
+import { useRatesBeer } from '../../api/hook/beers';
+import { IBeer } from '../../api/types/beers';
 
 const Main = () => {
-  const [activeTab, setActiveTab] = React.useState<string>('0');
+  const router = useRouter();
+  const homeTabs = { RATES: 'RATES', PREFERENCE: 'PREFERENCE' };
+  Object.freeze(homeTabs);
+
+  const [activeTab, setActiveTab] = React.useState<string>(homeTabs.RATES);
+  const { data: ratesData } = useRatesBeer();
 
-  const tabMenuHandle = (name: string): void => {
-    // if(name === "preferences" && !performance){
-    // TODO : 모달 띄워주기
-    // }
-    setActiveTab((prev: string) => name);
+  const tabMenuHandle = (tabName: string): void => {
+    // TODO : 유저 선호맥주 선택 여부 모달 띄워주기
+    setActiveTab(tabName);
   };
 
+  console.log(ratesData);
+
   return (
     <HomeLayout>
       <MainContainer>
         <MainTab>
           <TabButton
             activeTab={activeTab}
-            id="0"
+            tabName="RATES"
             onClick={() => {
-              tabMenuHandle('0');
+              tabMenuHandle('RATES');
```

Comment:
문자열 그대로 대신 만들어놓은 변수 사용해도 좋을것 같아요!
```suggestion
            tabName={homeTabs.RATES}
             onClick={() => {
              tabMenuHandle(homeTabs.RATES);
```